bindings_ffi/src/mls.rs

```diff
@@ -771,13 +771,13 @@ impl From<BackupMetadata> for FfiBackupMetadata {
 }
 
 #[derive(uniffi::Record)]
-pub struct FfiBackupOptions {
+pub struct FfiArchiveOptions {
     start_ns: Option<i64>,
     end_ns: Option<i64>,
     elements: Vec<FfiBackupElementSelection>,
 }
-impl From<FfiBackupOptions> for BackupOptions {
-    fn from(value: FfiBackupOptions) -> Self {
+impl From<FfiArchiveOptions> for BackupOptions {
```

Comment:
Do we need to change this too?
```suggestion
impl From<FfiArchiveOptions> for ArchiveOptions {
```

Reply:
It's a Proto. I'll update those later.